__test__/integration/debt_token_api/scenarios/scenarios.ts

```diff
@@ -2,13 +2,24 @@ import { SimpleInterestLoanOrder } from "src/adapters/simple_interest_loan_adapt
 import { BigNumber } from "bignumber.js";
 
 export namespace DebtTokenScenario {
-    export interface Scenario {
+    export interface BaseScenario {
         description: string;
+    }
+
+    export interface Scenario extends BaseScenario {
         orders: SimpleInterestLoanOrder[];
         creditor: string;
         debtor: string;
     }
 
+    export interface Throwable {
+        shouldSucceed: boolean;
+        errorType?: string;
+        errorMessage?: string;
+    }
+
```

Comment:
NIce ^^

Reply:
Right? I think we could apply a lot of what we've done here elsewhere. But that's for another day.